package.json

```diff
@@ -4,7 +4,8 @@
   "description": "The Core functionality of Doctor - The Cloud Elements Asset Management Tool",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "test:coverage": "jest --coverage"
```

Comment:
Can you also run a lint check here? `npx eslint src --fix && jest`

Reply:
@MohdRafiUddin Will open a separate PR for eslint as it will cause changes in almost all files